### DISCUSSION TITLE
hwlp

### DISCUSSION THREAD

**Discussion by ghost** [community/community#142134] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I dont now what happend
![Captura de pantalla 2024-10-21 105430](https://github.com/user-attachments/assets/e2ea2eb6-844e-4c22-ae46-89fd07b607ea)

**Comment by ebndev** [comment]
Hi @oscareox553,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
